var/spack/repos/builtin/packages/lzo/package.py

```diff
@@ -37,15 +37,8 @@ class Lzo(Package):
     version('2.06', '95380bd4081f85ef08c5209f4107e9f8')
     version('2.05', 'c67cda5fa191bab761c7cb06fe091e36')
 
-    def install(self, spec, prefix):
-        configure_args = [
-            '--prefix={0}'.format(prefix),
+    def configure_args(self):
+        return [
             '--disable-dependency-tracking',
             '--enable-shared'
         ]
-        configure(*configure_args)
-        make()
-        if self.run_tests:
-            make('check')
```

Comment:
@davydden In addition, only to be clearer : if a target is not found then `make(target)` is not run. For example trying to run:
```
spack install --run-tests  szip
``` 
you'll see at some point in the log files:
```
...
==> Trying default sanity checks [check]
==> Target 'test:' not found in Makefile
==> 'make' '-j4' 'check'
...
```